pyad/reverse_mode.py

```diff
@@ -146,3 +145,23 @@ def sqrt(x):
 
 def cbrt(x):
 	return _elementary_op(x, np.cbrt, lambda x: 1 / (3 * x**(2/3)))
+
+
+# Simple test suite explaining how Reverse works
+#Initialization
+x = Reverse(0.5)
+y = Reverse(4.2)
+z = Reverse(3)
+f = x * y**3 + sin(x) - logistic(z)
+
+#set df seed to be 1
+f.grad_value = 1.0
+
+f_val = f.value
+x_val = x.value
+x_grad = x.get_grad()
```

Comment:
Can we make the `get_grad()` function a `@property` instead of a function, and maybe rename it to `grad()`. So, the user could write

```
x_val = x.value
x_grad = x.grad
```